jest.config.js

```diff
@@ -9,10 +9,10 @@ module.exports = {
   transform: {
     '^.+\\.tsx?$': 'babel-jest',
   },
-  moduleDirectories: ['node_modules', 'server'],
-  moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, {
-    prefix: '<rootDir>/',
-  }),
+  // moduleDirectories: ['node_modules', 'server'],
+  // moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, {
+  //   prefix: '<rootDir>/',
+  // }),
```

Comment:
From personal experience, I can tell, that it is better to use a block comment + add a line stating the reason, why code is commented (instead of deleted).